### PR TITLE
pull OL docker image

### PR DESCRIPTION
**Pull request by DannyS03**
https://github.com/OpenLiberty/guides-common/issues/439

### CODE REVIEW

**Comment by TiagoF99** [comment]
verified on lgdev. Readme LGTM.

**Comment by DannyS03** [comment]
> @DannyS03
> 
>     * [x]  need to rewrite the followingparagraph in README.adoc and see any other places need to be updated
> 
>     * "In this case, you’re using the `open-liberty` image ..."

README updated.

**Comment by gkwan-ibm** [comment]
@Charlotte-Holt would you review the change in the README?

File: README.adoc [hunk 1]
```diff
@@ -126,11 +126,10 @@ inventory/Dockerfile
...
 
-In this case, you're using the `open-liberty` image as your parent
-image, which comes with the Open Liberty runtime by default. You can find all different official images at https://hub.docker.com/_/open-liberty[open-liberty Docker Hub^]. To pull a different open-liberty image, like `kernel`, define the `FROM` instruction as `FROM open-liberty:kernel`. If you build an image for production, it is recommended to use the `openliberty/open-liberty:kernel-java8-openj9-ubi` image.
+In this case, you are using the recommended production image, 
+`openliberty/open-liberty:kernel-java8-openj9-ubi`, as your parent image. If you would not like to have any additional runtime features for your `kernel` image, define the `FROM` instruction as `FROM open-liberty:kernel`. To use the default image that comes with the Open Liberty runtime, simply define the `FROM` instruction as `FROM open-liberty`. You can find all the different official images at https://hub.docker.com/_/open-liberty[open-liberty Docker Hub^].
```
Comment (Charlotte-Holt) [inline README.adoc:132]: Suggested update for this paragraph: 
>In this case, you're using the recommended production image, 
`openliberty/open-liberty:kernel-java8-openj9-ubi`, as your parent image. If you don't want any additional runtime features for your `kernel` image, define the `FROM` instruction as `FROM open-liberty:kernel`. To use the default image that comes with the Open Liberty runtime, define the `FROM` instruction as `FROM open-liberty`. You can find all the official images at https://hub.docker.com/_/open-liberty[open-liberty Docker Hub^].

Comment (DannyS03) [reply README.adoc]: Thanks @Charlotte-Holt. Updated the README.

**Comment by DannyS03** [comment]
Tested/verified on **qa**.

**Pull request merged**